ru/turing/courses/lesson5/Student.java

```diff
@@ -0,0 +1,158 @@
+package ru.turing.courses.lesson5;
+
+import java.util.Comparator;
+import java.util.Objects;
+
+/**
+ * Класс Student с тремя полями: id, name, phone
+ */
+public class Student {
```

Comment:
```suggestion
public class Student implements Comparable {
```
Если ты хочешь сделать прям красиво, я бы ещё вставил какую-нибудь сортировку по умолчанию через этот интерфейс